protocols/Phagelux_Canada/sterile_workflow.ot1.py

```diff
@@ -0,0 +1,75 @@
+from opentrons import containers, instruments
+import math
+
+# labware setup
+tiprack_10ul = containers.load('tiprack-10ul', 'A2')
+tiprack_200ul = containers.load('tiprack-200ul', 'C2')
+
+# 8 tubes are arranged in A1, B1, C1... D2
+sample_tubes = containers.load('tube-rack-2ml', 'A3')
+trough = containers.load('trough-12row', 'E1')
+plate = containers.load('96-flat', 'B2')
+
+tsb_media = trough.wells('A1')
+
+trash = containers.load('trash-box', 'B1')
+
+# Pipette setup
+m300 = instruments.Pipette(
+    axis='a',
+    max_volume=300,
+    channels=8,
+    tip_racks=[tiprack_200ul])
+
+p10 = instruments.Pipette(
+    axis='b',
+    max_volume=10,
+    channels=1,
+    tip_racks=[tiprack_10ul])
+
+
+def run_custom_protocol(diameter: int=60):
+
+    # Create new labware depending on diameter input
+    subdiameter = round(math.sqrt(diameter**2/2)/4, 1)
+    if 'petri-dish-4x4grid' not in containers.list():
```

Comment:
Pseudo-code:
-> if container does not exist, create it.
-> elif container does exist and diameter is different then delete current container
-> elif container does exist and diameter is same, do nothing.